### DISCUSSION TITLE
Having issues after Lazy update

### DISCUSSION THREAD

**Discussion by statquant** [R-nvim/R.nvim#177] [Q&A]
I just updated plugins with Lazy and I cannot get R.nvim to work anymore

I'm getting the following error:
![image](https://github.com/R-nvim/R.nvim/assets/3612473/eb2ea130-45c5-4093-8002-739958f68d92)

I use a few plugins
![image](https://github.com/R-nvim/R.nvim/assets/3612473/b91bb315-9c0d-47e4-9ac4-51fcc56811cb)

I use R-nvim main/6286b6b, tree-sitter master/a8c082b, cmp-R efa34e7
Would anyone have pointers to help diagnosing the issue ?

**Comment by PMassicotte** [comment]
Check in opened issues. There is one way to fix it by using a committed version of treesitter while waiting my PR to be merged.
